client/src/apps/code-collabo/components/AppPageStructure.tsx

```diff
@@ -36,13 +36,27 @@ export default function PageStructure({ children }: { children: ReactNode }) {
 
       {/* TODO: Convert sidebar into a component - inject one here, and one after 1st button in header */}
       {/* Sidebar for Desktop & left side menu for mobile */}
-      <div className='app__menubar__nav'>
-        <div><b>LOGO HERE</b></div>
-        <nav>
-          <Link href='/code-collabo'>Overview</Link>
-          <Link href='/code-collabo/projects'>Projects</Link>
-          <Link href='/code-collabo/careers'>Careers</Link>
-          <Link href='/code-collabo/donate'>Donate</Link>
+      <div className='app__menubar__nav lib__flex-center-col'>
+        <Link className='app__logo' href='/code-collabo'>
+          <Image src='/code-collabo/logo.png' alt='logo' width={207} height={55} />
+        </Link>
+        <nav className='app__menubar__nav__items lib__flex-space-btw-col'>
+          <Link className='app__menubar__nav__link lib__flex-center' href='/code-collabo'>
+            <Image src='/code-collabo/dashboard.png' alt='donate-icon' width='17' height='15' />
+            Overview
+          </Link>
+          <Link className='app__menubar__nav__link lib__flex-center' href='/code-collabo/projects'>
+            <Image src='/code-collabo/project-icon.png' alt='project-icon' width='17' height='15' />
+            Projects
```

Comment:
Projects link icon: Something changes the color of the icon to white or so on the page - so the icon is not visible, help look into this.

Reply:
it is whie that is why it is not visible. it is the Active state that is why. The designer will have to provide an inactive image icon for it since we are using the exported image icons for now.

Reply:
Oh... Ok, nice. Thank you. I totally forgot. Thank you. I've add it as a reminder comment for @Geefundesign on figma: https://www.figma.com/file/dyR55ZfOIQBvIJo6UMxZC9?node-id=47:13546&mode=design#731401524

You can remind him too (on figma - since that is his workspace, or on discord public channel).